src/utils/tripPattern.ts

```diff
@@ -44,6 +44,19 @@ export function isValidNonTransitDistance(
     )
 }
 
+export function createParseTripPattern(): (rawTripPattern: any) => TripPattern {
```

Comment:
Er ikke denne helt lik den fra `src/otp2/controller.ts`?

Reply:
Jo, men den returer en annen type TripPattern, så jeg lagde to forskjellige